Inconsistent ALIGNN tests

Tests for ALIGNN that call `get_figshare_model_ff` seem to still fail inconsistently with errors like:

```
tests/test_single_point.py:284: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
janus_core/calculations/single_point.py:186: in __init__
    self.set_calculator()
janus_core/calculations/single_point.py:213: in set_calculator
    calculator = choose_calculator(
janus_core/helpers/mlip_calculators.py:200: in choose_calculator
    path = default_path()
../../../.cache/pypoetry/virtualenvs/janus-core--tyG0Wny-py3.9/lib/python3.9/site-packages/alignn/ff/ff.py:129: in default_path
    dpath = get_figshare_model_ff(model_name="v5.27.2024")
../../../.cache/pypoetry/virtualenvs/janus-core--tyG0Wny-py3.9/lib/python3.9/site-packages/alignn/ff/ff.py:98: in get_figshare_model_ff
    zp = zipfile.ZipFile(path)
/opt/hostedtoolcache/Python/3.9.19/x64/lib/python3.9/zipfile.py:1268: in __init__
    self._RealGetContents()
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <zipfile.ZipFile [closed]>

    def _RealGetContents(self):
        """Read in the table of contents for the ZIP file."""
        fp = self.fp
        try:
            endrec = _EndRecData(fp)
        except OSError:
            raise BadZipFile("File is not a zip file")
        if not endrec:
>           raise BadZipFile("File is not a zip file")
E           zipfile.BadZipFile: File is not a zip file

/opt/hostedtoolcache/Python/3.9.19/x64/lib/python3.9/zipfile.py:1335: BadZipFile
----------------------------- Captured stdout call -----------------------------
dir_path /home/runner/.cache/pypoetry/virtualenvs/janus-core--tyG0Wny-py3.9/lib/python3.9/site-packages/alignn/ff/v5.27.[202](https://github.com/stfc/janus-core/actions/runs/10383480244/job/28748487868#step:6:203)4
----------------------------- Captured stderr call -----------------------------

  0%|          | 0.00/118 [00:00<?, ?iB/s]
100%|██████████| 118/118 [00:00<00:00, 415kiB/s]
```

Generally rerunning the jobs seems can get them to pass, but we don't want to have to do this for every PR going forward...

--- a/tests/test_mlip_calculators.py
+++ b/tests/test_mlip_calculators.py
@@ -1,6 +1,7 @@
 """Test configuration of MLIP calculators."""
 
 from pathlib import Path
+from zipfile import BadZipFile
 
 from chgnet.model.model import CHGNet
 from matgl import load_model
@@ -119,12 +120,15 @@ def test_invalid_device(arch):
 )
 def test_extra_mlips(arch, device, kwargs):
     """Test extra MLIPs (alignn) can be configured."""
-    calculator = choose_calculator(
-        arch=arch,
-        device=device,
-        **kwargs,
-    )
-    assert calculator.parameters["version"] is not None
+    try:
+        calculator = choose_calculator(
+            arch=arch,
+            device=device,
+            **kwargs,
+        )
+        assert calculator.parameters["version"] is not None
+    except BadZipFile:
+        pytest.skip()
 
 
 @pytest.mark.extra_mlips

--- a/tests/test_single_point.py
+++ b/tests/test_single_point.py
@@ -14,6 +14,7 @@ MODEL_PATH = Path(__file__).parent / "models"
 
 MACE_PATH = MODEL_PATH / "mace_mp_small.model"
 SEVENNET_PATH = MODEL_PATH / "sevennet_0.pth"
+ALIGNN_PATH = MODEL_PATH / "v5.27.2024"
 
 test_data = [
     (DATA_PATH / "benzene.xyz", -76.0605725422795, "energy", "energy", {}, None),
@@ -270,7 +271,12 @@ def test_mlips(arch, device, expected_energy):
 
 
 test_extra_mlips_data = [
-    ("alignn", "cpu", -11.148092269897461, {}),
+    (
+        "alignn",
+        "cpu",
+        -11.148092269897461,
+        {"model_path": ALIGNN_PATH / "best_model.pt"},
+    ),
     ("sevennet", "cpu", -27.061979293823242, {"model_path": SEVENNET_PATH}),
     ("sevennet", "cpu", -27.061979293823242, {}),
     ("sevennet", "cpu", -27.061979293823242, {"model_path": "SevenNet-0_11July2024"}),

